mentat/app.py

```diff
@@ -37,26 +36,7 @@ def run_cli():
     args = parser.parse_args()
     paths = args.paths
     exclude_paths = args.exclude
-    run(expand_paths(paths), expand_paths(exclude_paths))
-
-
-def expand_paths(paths: Iterable[str]) -> Iterable[str]:
```

Comment:
I want to keep the 'expand_paths' in app.py; the reason why is because some terminals (like zsh) automatically expand globbed paths. This means the later on we call this function, the more likely it is that we get some difference in functionality between terminals. It would definitely be good to add a comment mentioning this to this function though, since this isn't clear at all!

Reply:
Makes sense. Also thinking about it more one would only want this functionality in the terminal interface anyway. As an editor plugin it would probably be better to build context from opened files.